cg/meta/upload/scout/uploadscoutapi.py

```diff
@@ -272,6 +279,34 @@ def upload_fusion_report_to_scout(
 
         LOG.info(f"Upload {report_type} fusion report finished!")
 
+    def upload_rna_delivery_report_to_scout(self, dry_run: bool, case_id: str) -> None:
+        """Upload rna delivery report file for a case to Scout."""
+
+        rna_delivery_report: File | None = self.get_rna_delivery_report(case_id)
+        if not rna_delivery_report:
+            raise FileNotFoundError(
+                f"Rna delivery report was not found in Housekeeper for {case_id}."
+            )
+
+        LOG.info(f"Rna delivery report {rna_delivery_report.path} found")
+
+        related_dna_cases: set[str] = self.get_related_uploaded_dna_cases(case_id)
+        if not related_dna_cases:
+            raise CgDataError("No connected DNA case has been uploaded.")
+
+        for dna_case_id in related_dna_cases:
+            LOG.info(f"Uploading rna delivery report to Scout for case {dna_case_id}.")
+
+            if dry_run:
+                continue
+            self.scout_api.upload_rna_delivery_report(
+                case_id=dna_case_id,
+                report_path=rna_delivery_report.full_path,
+            )
+            LOG.info("Uploaded rna delivery report fusion report.")
```

Comment:
I would put the logging inside the for loop as DEBUG to avoid spamming the logs and the ones outside as info
```suggestion
            LOG.debug(f"Uploaded rna delivery report fusion report for {dna_case_id}.")
```